[2023-03-01 12:48:04,022] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-13T08:05:00+00:00 [queued]>
[2023-03-01 12:48:04,125] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_name 2022-11-13T08:05:00+00:00 [queued]>
[2023-03-01 12:48:04,130] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:48:04,136] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:48:04,137] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:48:04,263] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_name> on 2022-11-13T08:05:00+00:00
[2023-03-01 12:48:04,270] {standard_task_runner.py:52} INFO - Started process 13127 to run task
[2023-03-01 12:48:04,326] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_name', '2022-11-13T08:05:00+00:00', '--job-id', '3074', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmpbj4hn41i', '--error-file', '/tmp/tmp6_y5obhb']
[2023-03-01 12:48:04,359] {standard_task_runner.py:77} INFO - Job 3074: Subtask get_name
[2023-03-01 12:48:04,910] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_name 2022-11-13T08:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:48:05,623] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2022-11-13T08:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-13T08:05:00+00:00
[2023-03-01 12:48:05,767] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:48:05,910] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_name, execution_date=20221113T080500, start_date=20230301T124804, end_date=20230301T124805
[2023-03-01 12:48:06,380] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:48:06,492] {local_task_job.py:146} INFO - Task exited with return code 0
